scss/techs.scss

```diff
@@ -22,6 +28,8 @@ $card-width: 210px;
     left: -10px;
     position: absolute;
     top: -10px;
+    transition: left 0.5s,
```

Comment:
Arrange both properties on the same line.

Reply:
Done